bedrock/settings/static_media.py

```diff
@@ -1000,7 +1000,7 @@
     # bundle should offset the extra weight.
     'common': {
         'source_filenames': (
-            'js/libs/jquery-1.11.3.min.js',
+            'js/libs/jquery-1.11.3.min.js',  # can now upgrade to jQuery 3?
```

Comment:
We'll need to do a lot more testing with jQuery 3 to make sure there are no regressions as quite a lot has changed. But in theory this does indeed make it possible 👍